baseimage/Dockerfile

```diff
@@ -24,16 +23,22 @@ RUN mkdir -p /usr/share/maven /usr/share/maven/ref \
   && ln -s /usr/share/maven/bin/mvn /usr/bin/mvn
 
 FROM maven AS builder
-RUN  mkdir -p /opt/ol \
-  && mkdir -p /stacks/java-openliberty/starterapp
 
-RUN chmod -R 777 /stacks/java-openliberty/starterapp
+RUN  mkdir -p /mvn/repository \
+  && chown -R java_user /mvn \
+  && mkdir -p /stacks/java-openliberty/priming-app \
+  && chown -R java_user /stacks 
 
 COPY ./LICENSE /licenses/
-ADD ./starterapp/src /stacks/java-openliberty/starterapp/src
-COPY ./starterapp/pom.xml /stacks/java-openliberty/starterapp/
 
-WORKDIR /stacks/java-openliberty/starterapp
+USER java_user
+
+RUN chmod -R 755 /stacks/java-openliberty/priming-app
```

Comment:
Don't think we need this since java_user already owns this.  But this is an empty dir, so probably not worth respinning.